dotnet/src/IntegrationTests/Connectors/Amazon/Bedrock/BedrockChatCompletionTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Amazon;
+
+public class BedrockChatCompletionTests
+{
+    [Theory]
+    [InlineData("ai21.jamba-instruct-v1:0")]
+    [InlineData("amazon.titan-text-premier-v1:0")]
+    [InlineData("amazon.titan-text-lite-v1")]
+    [InlineData("amazon.titan-text-express-v1")]
+    [InlineData("anthropic.claude-v2")]
+    [InlineData("anthropic.claude-v2:1")]
+    [InlineData("anthropic.claude-instant-v1")]
+    [InlineData("anthropic.claude-3-sonnet-20240229-v1:0")]
+    [InlineData("anthropic.claude-3-haiku-20240307-v1:0")]
+    [InlineData("cohere.command-r-v1:0")]
+    [InlineData("cohere.command-r-plus-v1:0")]
+    [InlineData("meta.llama3-70b-instruct-v1:0")]
+    [InlineData("meta.llama3-8b-instruct-v1:0")]
+    [InlineData("mistral.mistral-7b-instruct-v0:2")]
+    [InlineData("mistral.mistral-large-2402-v1:0")]
+    [InlineData("mistral.mistral-small-2402-v1:0")]
+    [InlineData("mistral.mixtral-8x7b-instruct-v0:1")]
+    public async Task ChatCompletionReturnsValidResponseAsync(string modelId)
```

Comment:
A couple of points here:
1. The authentication should be configured to run these tests in merge pipeline.
2. Running the test for each model during merge pipeline every time a new PR is merged looks scary, I think there will be a lot of transient errors. I also think there could be an issue with rate limiting. 

I would probably disable these tests by default and run them manually when needed. Later we can setup a job to run them periodically and independently from PR merge pipeline.